release-notes/con-relnotes-techpreview-features.adoc

```diff
@@ -58,10 +61,22 @@ Additional detail on how Red Hat provides support for bundled community dynamic
 
 |Azure Devops|`@backstage/plugin-azure-devops-backend`|Azure DevOps backend plugin that contains the API for retrieving builds and pull requests, that are used by the Azure DevOps frontend plugin.
 
+|Azure Devops Scaffolder|`@backstage/plugin-scaffolder-backend-module-azure`|The azure module for @backstage/plugin-scaffolder-backend.
+
+|Bitbucket Cloud|@backstage/plugin-catalog-backend-module-bitbucket-cloud|A Backstage catalog backend module that helps integrate towards Bitbucket Cloud.
+
+|Bitbucket Cloud Scaffolder|@backstage/plugin-scaffolder-backend-module-bitbucket-cloud|The Bitbucket Cloud module for @backstage/plugin-scaffolder-backend.
+
+|Bitbucket Server|@backstage/plugin-catalog-backend-module-bitbucket-server|A Backstage catalog backend module that helps integrate towards Bitbucket Server.
```

Comment:
For this one the required_variables is: `BITBUCKET_WORKSPACE`

cf https://github.com/janus-idp/backstage-showcase/blob/1.1.x/dynamic-plugins.default.yaml#L622

Reply:
There was a copy/paste error here: it is `BITBUCKET_HOST`

Reply:
fixed in suggestions elsewhere in this PR. Please review and merge the suggestions if they're correct.